build.sh

```diff
@@ -1,33 +1,29 @@
 #!/usr/bin/env bash
 set -e
 
-npm_package_version="${npm_package_version?Script must be run using npm}"
+declare -a php_versions=(
+	7.2
+	7.1
+	7.0
+	5.6
+)
+declare npm_package_version="${npm_package_version?Script must be run using npm}"
+declare dockerfile_dir
+dockerfile_dir="$(dirname "${BASH_SOURCE[0]}")"
 
 docker login
 
 # NOTE: Not building this stack of images concurrently due to a known issue
 # with docker concurrent builds. https://github.com/moby/moby/issues/9656
-
-docker build \
-  -t "visiblevc/wordpress:latest" \
-  -t "visiblevc/wordpress:latest-php7.2" \
-  -t "visiblevc/wordpress:$npm_package_version-php7.2" \
-./php7.2/
-
-docker build \
-  -t "visiblevc/wordpress:latest-php7.1" \
-  -t "visiblevc/wordpress:$npm_package_version-php7.1" \
-./php7.1/
-
-docker build \
-  -t "visiblevc/wordpress:latest-php7.0" \
-  -t "visiblevc/wordpress:$npm_package_version-php7.0" \
-./php7.0/
-
-docker build \
-  -t "visiblevc/wordpress:latest-php5.6" \
-  -t "visiblevc/wordpress:$npm_package_version-php5.6" \
-./php5.6/
+for php_version in "${php_versions[@]}"; do
+	docker build \
+		--build-arg PHP_VERSION="$php_version" \
+        --build-arg VERSION="$npm_package_version" \
+		-t "visiblevc/wordpress:latest" \
+		-t "visiblevc/wordpress:latest-php7.2" \
```

Comment:
I think this won't work, shouldn't it use the `$php_version` version? Also we don't want to set `latest` for all version, only for `7.2`

Reply:
It should work I think unless I'm overlooking something.

The `PHP_VERSION` arg is the one that sets the base image in the Dockerfile. 

The `npm_package_version` arg is only used for setting the version label in the Dockerfile.

---

I see what you mean Re: your `latest` concern. I'll add a fix for that so that only the latest php version image gets tagged with `latest`.

Reply:
Oh hah... Now I'm also noticing that `7.2` is hard-coded in there still.. Fixing that too :blush: